common/PacketTypes.acn

```diff
@@ -0,0 +1,28 @@
+PacketTypes DEFINITIONS ::= BEGIN
+
+ApplicationProcess-ID-Packet [size 11]
+SecondaryHeaderFlag [false-value '0'B]
+
+TM-Packet-ID []
+{
+    packetType [pattern '0'B],
+    secondaryHeaderFlag [],
+    applicationProcess-ID []
+}
+
+TC-Packet-ID []
+{
+    packetType [pattern '1'B],
+    secondaryHeaderFlag [],
+    applicationProcess-ID []
+}
+
+PacketSequenceControl []
+{
+    sequenceFlags [pattern '11'B],
+    packetSequenceCount [size 14]
+}
+
+PacketVersionNumber [pattern '000'B]
```

Comment:
isn't it something different in PUS-C?

Reply:
Seems like only PUS version changed, CCSDS packet version is the same

Reply:
"The packet version number is set to 0 and identifies it as a space packet defined by
CCSDS 133. 0-B-1. A space packet is also referred to as a version 1 CCSDS
packet"

Reply:
Ok, I might mix CCSDS version and PUS-C version - I never remember which field contains which :)